tests/test-notebooks/test_notebooks.py

```diff
@@ -0,0 +1,49 @@
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+
+import nbformat
+from nbconvert.preprocessors import ExecutePreprocessor
+from nbconvert.preprocessors.execute import CellExecutionError
+
+import glob
+import traceback
+import unittest
+import sys
+import time
+import os
+
+
+class TestNotebooks(unittest.TestCase):
+  def _execNotebook(self, ep, notebook_filename, nbpath):
```

Comment:
all other unit tests have a blank line between the first class method and the `class` line. also we use snake case, `test_notebooks` and `_exec_notebook`.